packages/material-ui/src/TableRow/TableRow.js

```diff
@@ -15,13 +16,13 @@ export const styles = theme => ({
     '&$selected': {
       backgroundColor:
         theme.palette.type === 'light'
-          ? 'rgba(0, 0, 0, 0.04)' // grey[100]
+          ? fade(theme.palette.primary.main, 0.04)
           : 'rgba(255, 255, 255, 0.08)',
     },
     '&$hover:hover': {
       backgroundColor:
         theme.palette.type === 'light'
-          ? 'rgba(0, 0, 0, 0.07)' // grey[200]
+          ? 'rgba(0, 0, 0, 0.04)'
```

Comment:
What if we were using `theme.palette.action.hover` instead?

Reply:
`theme.palette.action.hover` has 0.08 alpha, which does not match MDC (which is 0.04). Which one are we going to use?

Reply:
> `theme.palette.action.hover` has 0.08 alpha, which does not match MDC (which is 0.04). Which one are we going to use?

This is ok for now. `theme.palette.action.hover` will either be changed to match MDC or it's up to the app dev to strictly follow MDC here. 

This is one of the freedoms we grant ourselves. I think we somewhat agreed that 0.04 is not enough to visualize the state change.

Reply:
@eps1lon I think 0.04 is really fine but as you mentioned, devs can overwrite if they want to match MDC. I'll update it. btw do you have any consideration regarding the selected state color?

Reply:
> I think 0.04 is really fine but as you mentioned

I don't think it's fine. I think 0.08 is fine and this hover style should be consistent in a theme. If devs want a strict MDC implementation they can lover the hover opacity to 0.04.